pocs/camera/__init__.py

```diff
@@ -151,9 +151,6 @@ def kwargs_or_config(item, default=None):
         try:
             module = load_module('pocs.camera.{}'.format(camera_model))
             logger.debug('Camera module: {}'.format(module))
-        except ImportError:
```

Comment:
This seems like a very different type of problem from "the camera is connected but unresponsive" or "the camera is disconnected". ImportError seems to say that the config is bad. Why is that not worth raising for?

Reply:
I originally had it with both types of exceptions raised but we want the same behavior in either case,  namely to give a warning but still try the other cameras. The `logger.error` will display separate messages depending on why it failed. Maybe just a more generic message instead of "Cannot find camera type"?